### PR TITLE
Feature/handle expired cert

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

**Comment by miltontony** [comment]
image

**Comment by miltontony** [comment]
@Mitso ready for review

**Review by Mitso — APPROVED** [review]
🍰 To me.

File: higher_health/views.py [hunk 1]
```diff
@@ -42,6 +43,17 @@ def get_context_data(self, **kwargs):
         )
         return ctx
 
+    def dispatch(self, request, *args, **kwargs):
+        if self.request.user.is_authenticated and self.request.method == "GET":
+            triage = Covid19Triage.objects.filter(user=self.request.user).last()
+            if (
+                triage
+                and triage.timestamp.date() >= datetime.now().date()
+                and "redo" not in self.request.GET
```
Comment (rudigiesler) [inline higher_health/views.py:52]: I don't see anywhere in the templates where we have a `redo` link. Is this something that's just for testing?

**Review by rudigiesler — APPROVED** [review]
Just a question from me, looks good otherwise

File: higher_health/views.py (hunk 1 above)
Comment (miltontony) [reply higher_health/views.py:52]: No, cross PR - it's added in #91

Comment (miltontony) [reply higher_health/views.py:52]: @rudigiesler Let me add a test for it though

**Review by rudigiesler — APPROVED** [review]

**Pull request merged**